.test-infra/jenkins/job_PreCommit_Portable_Python.groovy

```diff
@@ -40,15 +42,17 @@ builder.build {
   // Due to BEAM-7993, run multiple Python version of portable precommit
   // tests in parallel could lead python3 container crash. We manually
   // config gradle steps here to run tests in sequential.
+  def lowestSupported = LOWEST_SUPPORTED.replace('.', '')
+  def highestSupported = HIGHEST_SUPPORTED.replace('.', '')
   steps {
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
-      tasks(':sdks:python:test-suites:portable:py37:preCommitPy37')
+      tasks(':sdks:python:test-suites:portable:py${lowestSupported}:preCommitPy${lowestSupported}')
       commonJobProperties.setGradleSwitches(delegate)
     }
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
-      tasks(':sdks:python:test-suites:portable:py310:preCommitPy310')
+      tasks(':sdks:python:test-suites:portable:${highestSupported}:preCommitPy${highestSupported}')
```

Comment:
Looks like these changes are causing [Python Portable Precommit to fail](https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Commit/27558/console) with:
```
Project 'py${lowestSupported}' not found in project ':sdks:python:test-suites:portable'
```
Need to use double quotes when using embedded values

Reply:
Fix in #26151